server/lib/pencil_space_server/game_monitor.ex

```diff
@@ -22,8 +21,24 @@ defmodule PencilSpaceServer.GameMonitor do
     end
   end
 
+  def add_host(name, host) do
+    GenServer.cast(ref(name), host)
```

Comment:
Instead of using `host` I would make this an explicit tuple (like the one you are sending `{:host, host}`)

That prevents calling this API with improper data. Currently I can potentially override the `host` variable to hit all the handle_cast callbacks. That should be restricted.

Reply:
Addressed.